### PR TITLE
MIlkyway API Documentation.

### PR DESCRIPTION
**Pull request by enecosmin2001**
- [x] Create slate documentation for MIlkyway API.
- [ ] Style and arrange the content
- [ ] Remove unnecessary content

### CODE REVIEW

File: source/includes/_milkyway_api_doc.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```shell
+curl --header "Content-Type: application/json" \
+ --header "Authorization: Bearer <ACCESS_TOKEN>"\
+ --request PUT \
+ --data '{"skills":["<SKILL_1>", "<SKILL_2>", "<SKILL_3>", "<SKILL_4>", "<SKILL_5>"]}' \
+ http://milkyway.sortlist.com/v1/<ENVIRONMENT>/entities/<ENTITY_ID>
+```
+>Example of request:
+
+```json
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]:
```suggestion
```shell
```

File: source/includes/_milkyway_api_doc.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+{ 
+  "id" : "cluj-napoca-studio",
+  "skills" : ["design", "frontend development", "angularJS", "reactJS", "html", "css", "web-development",] 
+}
+```
+
+
+>The above command returns JSON structured like this:
+
+```json
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]:
```suggestion
```shell
```

File: source/includes/_milkyway_api_doc.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ --request PUT \
+ --data '{"skills":["<SKILL_1>", "<SKILL_2>", "<SKILL_3>", "<SKILL_4>", "<SKILL_5>"]}' \
+ http://milkyway.sortlist.com/v1/<ENVIRONMENT>/entities/<ENTITY_ID>
+```
+>Example of request:
+
+```json
+{ 
+  "id" : "cluj-napoca-studio",
+  "skills" : ["design", "frontend development", "angularJS", "reactJS", "html", "css", "web-development",]
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]:
```suggestion
  "skills" : ["design", "frontend development", "angularJS", "reactJS", "html", "css", "web-development"] 
```

File: source/includes/_milkyway_api_doc.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+>Shell command:
+
+```shell
+curl --header "Content-Type: application/json" \
+ --header "Authorization: Bearer <ACCESS_TOKEN>"\
+ --request PUT \
+ --data '{"skills":["<SKILL_1>", "<SKILL_2>", "<SKILL_3>", "<SKILL_4>", "<SKILL_5>"]}' \
+ http://milkyway.sortlist.com/v1/<ENVIRONMENT>/entities/<ENTITY_ID>
+```
+>Example of request:
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]: there is two times an example of request, keep only one of them

File: source/includes/_milkyway_api_doc.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+This entry represents the **link between an entity and a input skill**.
+
+* Create a new entry (entity_skill, milkyway_skill) (called translation) in table `mw_entity_skill_translation`:
+  1. **entity_skill** : agency skill slug (input skill)
+  2. **milkyway_skill** : is the best match of the input skill in the MW skills space. This is done using fuzzy matching.
+      * `input skill`		<=>	 	`milkyway_skill`
+      * (database relation :  many (*) to one (1) )
+
+
+This entry represents the **link between a input skill and a milkyway skill**.
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]:
```suggestion
This entry represents the **link between a entity skill and a milkyway skill**.
```

Is not an entity skill?

File: source/includes/_milkyway_api_doc.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+### Http request
+
+`PUT http://milkyway.sortlist.com/v1/<ENVIRONMENT>/entities/<ENTITY_ID>`
+
+### Query Parameters
+
+Parameter | Necessary | Type | Description
+--------- | -------- | ------- | -----------
+**`id`** | `no` | `string` | `ID (slug) of the entity`
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]:
```suggestion
**`ENTITY_ID`** | `no` | `string` | `ID (slug) of the entity`
```

File: source/includes/_milkyway_api_doc.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      * (database relation :  many (*) to one (1) )
+
+
+This entry represents the **link between a input skill and a milkyway skill**.
+
+### Http request
+
+`PUT http://milkyway.sortlist.com/v1/<ENVIRONMENT>/entities/<ENTITY_ID>`
+
+### Query Parameters
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md:139]: missing ENVIRONMENT parameter

File: source/index.html.md [hunk 8]
```diff
@@ -3,237 +3,15 @@ title: API Reference
 
 language_tabs: # must be one of https://git.io/vQNgJ
   - shell
-  - ruby
-  - python
-  - javascript
+  - json
```
Comment (charlesdg) [inline source/index.html.md]: remove it

File: source/includes/_milkyway_api_doc.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+Parameter | Necessary | Type | Description
+--------- | -------- | ------- | -----------
+**`id`** | `no` | `string` | `ID (slug) of the entity`
+**`skills`** | `yes` | `list` | `List of skills saved for entity`
+
+<aside class="warning">
+Braer token needed for authentification!
```
Comment (charlesdg) [inline source/includes/_milkyway_api_doc.md]:
```suggestion
Bearer token needed for authentification!
```

**Review by charlesdg — CHANGES_REQUESTED** [review]
Put the description at the end maybe

File: source/includes/_milkyway_api_doc.md (hunk 4 above)
Comment (enecosmin2001) [reply source/includes/_milkyway_api_doc.md]: i feel like is better to put the http request and the curl command separately

**Review by charlesdg — APPROVED** [review]

**Pull request merged**